### PR TITLE
[Tidy] Replace `dmc.Tabs` with `dbc.Tabs`

### PR DESCRIPTION
**Pull request by huong-li-nguyen**
## Description

We initially used `dmc.Tabs` due to an issue with `dbc.Tabs` at the time of development. Specifically, Plotly charts within `dbc.Tabs `did not resize correctly, requiring either a fixed height for the charts or the use of the callback method for the tabs, which we wanted to avoid. For more context on the previous issue, see:

- https://github.com/McK-Internal/vizro-internal/issues/292
- https://github.com/facultyai/dash-bootstrap-components/issues/211

I took another look at `dbc.Tabs`, and it seems to be working great now 🥳. 

It did take a bit of work to get everything running smoothly because the HTML structure for the Bootstrap component is very different. Plus, `dbc` adds an extra `div` for some reason that we can't assign a `className` to. This led to a couple of CSS issues, which are now sorted out. I left a comment about it, but we shouldn't need to change it anymore.

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-core/src/vizro/static/css/bootstrap_overwrites.css [hunk 1]
```diff
@@ -36,10 +37,32 @@ All the HEX values starting with --text-code are taken from the Github code high
...
   border-left: 2px solid var(--border-enabled);
 }
+
+/* TABS */
+.nav-tabs {
+  margin-bottom: 1.25rem;
+}
+
+.tab-content {
+  height: calc(100% - 3.25rem);
```
Comment (pruthvip15) [inline vizro-core/src/vizro/static/css/bootstrap_overwrites.css:55]: nit picking: lets just a comment explaining how 3.25 came here: header height + gap etc

**Review by pruthvip15 — APPROVED** [review]
One small comment, but looks good to my understanding.
Amazing work 👍

**Review by maxschulz-COL — APPROVED** [review]
Lgtm! Seems like a super small change, but I bet it wasn't :P 

One QQ: if we follow the recommended practice of adding a title to the `Graph` model, don't you think the title margin is a little small:

image

**Comment by huong-li-nguyen** [comment]
> Lgtm! Seems like a super small change, but I bet it wasn't :P
> 
> One QQ: if we follow the recommended practice of adding a title to the `Graph` model, don't you think the title margin is a little small:

Can you open it in an incognito browser? I think your browser might have cached an old CSS file? In your screenshot there is no styling applied to the tabs. 

It should look like this:

Screenshot 2024-11-26 at 11 32 33

**Comment by maxschulz-COL** [comment]
> Can you open it in an incognito browser? I think your browser might have cached an old CSS file? In your screenshot there is no styling applied to the tabs.

I get this in incognito - this is the link to [PyCafe](https://py.cafe/snippet/vizro/v1?pycafe-app-view=true&pycafe-edit-enable=true#c=H4sIAAeoRWcEA-2c3XIbR3aAX2UCe72URQ7n_0cxtyJTluk1ZXu9XkYrQ0UPgCE5FjjAzgwoUl5vbhO_Ru6cqzxAKhfRRSqVymVeI6-Q7_TPAOCPaG_WiVIraDDo093n__TpMyO1vhmMZ5NycG8wVH8elGdOMZ873czpmgunO6nq49aZLTpXjw_rYV2dzmdN55xVL5qZewr2tHWK1jk7vTQ0n8666YVbns-bslVT5ufD-qiZneoJjpl9IIAQPi7mp1U9KZvDwPNSZ4f57qToCrcf2Ljj_m5RNhcbw8FFWTTOzo4jU4eDO4I_L45LsM5O3c9obQxrh09XddNyZzj4ohi1w8Gm7hzPYF2XddfufKl75AOizDKI9tPRtTrLfpi9O6u7oqqR6-qwfFZ4Ox_1zC9_rhfm8gd2HzbF_OQGVvZjWaq5jn8jU_s5qo4XTbmDpUfFTWqsfnpXKB_dQlw-t884R9wxlsSQdXerwPK5AGNaHZUfnM9_0PzxbDprfhSXO68Yf9XYH-Wl4FZxVrw0O7-FtnzeeOnGsafX9F8398cu7_-T9R3eyNR-lpHTjouuu1GX1c9PFD3Hk_lnZTMuflg0_NjoaasXYpf57IdNf32CbXWOHbN9Il0zmxI4hMfDair-Q_LFab0u-h1BUHvgpGhPRrOimeiN8IEFN2RzhI78PFUzh7XaeNlSR4tqOtnoMe-4zaLeYM5gc9CUv1tUTXkq0UuRcNJ18_be9vb8wh0XR-X28cl20XTVUTHutk_Hz6q6LS-21ea-TVc1q9ttaLXbvp_HURKGcehtz6u5nrLlub4bZO6kPPO25hfhVs0q2SrqC_f5yRTe3cVcChM1F3B-0Z3MajrmF7NJNSm3zjw3SFyfoWlxQYkyuFcvptPNwVE1LRH2y28GdXEqFLaLti27dnvc8l203ezUpQneopkybJVqiufucdWdLEaLljjFumjtsnovqxZPJpPYK_LCS8e-P_LTNEz8LB2VyThLj3LPHxVpkQWlUXM8a8rt8rw4nSPXdjtuim58cojWN8o1a6rjqi6mkoCQARnfEr9tnZQFq9L5RoJjXkwmFGdb0_Kou-dk8_O_HNbf4rNvN6_ofVScVajjcnt9lF4X6qrG__XP__TZJx8OGwJ14Xl-oX89b-3efLT34PO1Hrn7mytA3zVSjWSl09wvvoXXYrXH3Lv53m_btR65Kzp-uNJztd9Xdw-6XKo_0z1X7n77-Yfva3hl9tg01qbqe3R8_9H9FXgFzRvRKFQr1mMr95ixjx7c_-JcQT7Qf_4dN4H88QfqJ1JDQnDY7XP_0I5757QefKyGg4p2zldff8P3-MFD7tf55839lXesJrHXX9Lrj_6q-7Lv0dfZx9z0oO7Q1wv90368pxtc__6P3HSo-0lEe-dgFU_aXvIura_fevzZcEjj8anq1L4fDWszyxdkrq_u_4q7oKtef3PDgInpUoH1BV8uCQN9_duqHJrer64RJbhHC1EEvkYY7n5g5i4F-pz7ikCivKyXNYE6vlwikMP3P1bFeZv2NbIY-qqd6nZ_qcmXO3f6lr7CvqWvZ3wVNRFIX-LtS139Uo_v646271B3WalczxSwahhtVsN02Om-zbcPBDIzNocDBdpBZ3Vw_53Vsf01xP11xH2NKG1P5VY_k_zA9dZjCeLHTyVAv9ITaOnrK77ShcCiroD6-lpNFMW5tEa6reJAYKW6n6q7MYDqN_q7zkFrRWVoTXSaMmokV_exnrDW5Rw8W8G4SuGZnQ5klpeCw03JhSKoIzfT_ouV9nDweG6aOrBBs_7-SP-oLuM5sRxXqplbQ326Dj7iy6X6vE3bEgrvScvGLte7asSE6pb-kTx-IqwbNRj0BNZMS0Os8K7BHTb691O-JxKa2mvr3O_1qFwvv1f9YjEumcclvDeEt0a_mffnL783mMMj_SucN4TzC4WwzvkXPSKX4SySDwcrSu8JY419M-MvXn6vEZXG7_PdMytxJSBUQ7yhGv6m2rL98O4ffi4u_tnB79-W37cPfv-BzBkJcy75GY6k6_yBau5KU5LScCQ8JJTMshB0ycujO3IT6p6YQUsg2-1oW4EmsQ1lxYwke3KplCzIXMp4GqvQPaK4zutc9ncu-L9RbKwNv7trQ_5f_lb9xn-tfsLect7dh8YT7s_Vghk8Hg5_aUczMba-HouNv_qdElbrYlwrvLRw1tfSVr3v6A5JH6PVQdUhLbWGRuIqTUGBqqX6fsaNqxTFvtbtj19-p1241HFjHdxbB096UIwtgK8zkW9NQ3c_8quVTt0jjUt94d2-1dsxvBuoX2_z5Xc6owlLcTJBqCbQ_NpkWLOnqv7AIMqeyPXyOzOg8NWIJqLwJeK4PtE_atS7q9OD2FQaXvLyO9WjCGR8zYDi8O61EmgKN0tgqCgC14qgcsZShODld66mcFeEt_NEhV9eK4DGvyrAXdVvaCj0a9mvcVehN6z_0HeaqabHpArVIzfpE8q2TyWhtR5u78hSFkDGJCRUiug73nvnsSx9A_z8sTTUJq97aIi6CjA7_F2VJrbUHqVaenNWTedA9pqvHr1jft82v0zRDSaovKZIFcagkjeE04pgXqQqjFUjyOpVQ7rg8OKxyjpmtDeIlV1t4qpHSP9MKWa3cG31TDI6e7fwkLT3dt8yfWrWxHRrQFUeAvLtLcB3tV_256-kGpOC-PF9u39oQKiYUkN1yCpcqNZ73Llkht2fLWij2W7SFpR-vVMrDGUivVUvYdmrpeZcohG0__r3wlPguqlU30gE4fr9E4WeSB0gaGd8LdI__AZA4LrR3lE5RF8aLTLlQ38ZVA2IOFzD-us7qt_yfPFLhax46qsPEYtqKnpQte9WOGvkV3B-q9f1xUeq1_L9g35WX-ErEi4R7_b6qnlrz0pv7q--a8sd6h_zaHLCd3XO6v2jDz6R_Pi-LKDrXhpVp_LCcJu_DHTbs-PX573RFbmuvjp6jwHneTXpTnhL6vMOezhwTsrq-KQDThR4VpXP35_JW2nP8RyZ4-gBXh5O6ZVXkUDnp9O6BRStUfr58-fu89CdNcfbYHjb8BkOfjGs32vKcXcLR0N4NC3Gz4aDbcGaF92JM6HzUZT7buQ7kZ-4abIfRbzZ5C9AgzR1Az_di_jNvWQ_9kM3jCMnzAI35REsjjI384J-Xpwlbuz5-3EYu6ReQ25PE3-ylOH5SdWh3SUZkiB2Mz9ygjh24zjaTfzcDaLcwk4C98wHDN04izFX7kZ54jDFTaJkN_FiN0TmKHZTL2MYWUUHdEnp7-EwApbpBuYFcBpC3aAHMIlluqEe8JLYY7phjkZ-wrCVzY9cP4eblV3Du0kYQC2zsJOEqZsGIbDBj3zYLKlH0g265Y5hwwB2VjgLW-EtbJWz-FZ5Q96axnK3prPSWcta6TX85FHipTg2Mw48sP5NIjFCfmD8ambd7tgkd-PcEgszhMmC_VRkTzIHJ7hemO8lqedGYdQzSzN0iMKD0EO3PN5P8YiKvcTNw4zh0A3iXhTF4lZJMh8eXmhFsayyKHCjNN7PMF8W-4hEbGTBbhZ7bu5jxzzCXrmDa0Aj3lEhiAMnSxLmYUeZm8asGWyYsgYS38kybOkTISk2Y2lleery9w8WPgh91AuCvUxoeSn4OeOO6XayNHCDPAUOXagKJ59oDf0YSXMky5EM4_msTH4zPOdFgRMGgeuHArPk8HBIAPkh8wkYn5UaEkC-SEPARHg8FM2zdAkTIX6aWbMae91q1xxD5jl2lXgmPvMkgiwu17CTx6KNxBO-hn1OfCJtBPksDnbzkFAImO5FbpKETi5a5LET5mhNPz4g7tAGmwbxbk4kJHSHKcoQzqQmN2aVkNeJoCWITnmWM93ARH0cpYzncMMWrGGsD_XE9XLI0e952a5wJ-PhdZQSeqFPNsPJOIhgENlTBWMqkQbdYmIAv-eiKqqnKcPG73mauUHa-303JxnG-XLcp8aT1WHxfZ4JsFVPHq_FrO2eve-FobsUDhAtlsL7HloHSa8cMIkMUxrle9gYp4e17QA1ujFtT15b3jC3frGyWb9Z2a1flW7o2Lsd3UNAGxXrUfOkDyPWFPsNnmBthNjSwOCzpiAg0kREOfTFVUibkaxFeawAH3DAC0SeACugHPKQanq4V97AaOUrV2h0AkAsZYgTfCkhbXl7mZsQTSJblGA49g0WpnWzBlkAJG5P3KJHESAir_fIkojZIC1t5EmFlmKMW8kq4GqpetAI3cNGqR7ZKN3TNkbpeRujiWgJTuuNum70W9e6JGDZGX0SVu5FNlX4xFES7V8aXaH2VhDE_sOHhp6ULvxeV_hNZ8ezw0nRPHu96r7LYt1W9lGfoXxfhMUKvFT2McfRA8ZIP7LsW3olIlhDidY43g9jtiXJUAGboh_shbmUc-F-RHh5FDFhwHaWpfsRkZJL6jHz2AiIKJ_yjxTFNgetPUN3xY_XR0VM5UKxQ4HjsZv7u0SgLHUDOpBnZcGKBUB6iflhL2e_lDpyN-Y3k5zn-ew-Mhy4eRo5Pnkm9VZhQtzLmW9gQjr3A8Y1vk_JEC_Js5G5fhwii2buk9QSVpSRzWeLzSLHSq7B3Rhj-CwfO8oizAMZNbiUUgnbeE9bbCXYljWrMxLeVjQLW9F72Khm8HvVDf3eNIa_NZyRrrfrutmfPIqlmpLNLY4PrF9jcnjkUXnhTzN-uz8T2dWIFehIEcGut5_wrx0kXCS8wr04IQ9Rrhkm5H8MkB2oGjLlGSDHB1KxSAWSyTgK5ZRzIoQmfqsQCXtegNFXlcEorheH-2nArkwalJqWvWU3ZSuiPqXmjRA2dlJ2bXblECnYalJMEMsujTkzP9xNI7J3JuU9qmGuFJ1UZYeKEW5IE0KEncHABwFJNAn9vTTGDdluSnmFBx3T7aQwzWTTZxdNg1SxI_UDi_d2U7jSBci-6YcOtY56jgqoWiUFp3Ahf1ID8AzixegimR9dMuoiqjbRNaYeCkjRBMISpgigWFQ2Nba61aYZoonGIZp4lGpZQE0rlaKGnYx0nlFnQJMdP3QyVZ4zX_J7lu6m8IykFkNln0eQFL6oz26FpUhAaRZRiLA9oRM-34U2_VLYYmHRHR0Tdh82xoBuAwY5BWKcMFuD7B4Yh1Hchz0D3JGxGoV4IG5CWDbvXWGeUyUHUt7ic2RLxA08WoTUWCJ7QBUjj00Z5UsmdQhbpsRSQCRneF9WkfV6Bp4vtZqGqbKpQ3uQIpy0Q2j16KSIWBaEJc9jMQX1kj15l5eoVrpMKohsKTxbEGloqRzbJSuFqDDK97CxjYWt6Sy-taylby1v2VvPWPGs56z41rOiXoqavecJcp5alpGxHjlPlqGEPag61EMGyBZWDyWJVBYIlPFsw95CiSuwR_jyyEQlQl5FAVGMcSoYHrVQkFTBU62FrQF6GAMRKNDT-IE8OZIELf2AVOOnyGP4B-qxjXHkI1KDOFJ6997W8G5GxohTmW7G2SZ9nvJ6dKInJxv15OVRTVa1ZU-Kzni47MWzsBXfwL16Br9X39DvzWP49-ZDPnmo6s27bv5bVz7Fp-NHOlvwPG3AFbQfV6FNpbBxmfSalWircv251mjXv-S7VCy8qdGsaBa2ovfw61Kj3eBPXUb9b9Vo1wvxpkb7n9Ro19u0z-x6p31To-E-6_V-19bwmxrtTY32_7JGu37l_wlqtIvidHp4VjYtpx_U31LOL16DCu16qa7WZ_wDJP58oEnIKdD2ZPbc6Y-GyIEUdaKokMMdTjsvx9VRVU7kcGfh_Pb-o30xVLlyWFQd9xQDT6uRPfD5GaCMGXBeyKkVdT50suzUh0ermeqvOB9qBpSmN5wtNVNE2VccNF0OuKdFzd8mN62dIadND03n-kR9zNVM60_ViBarOF8OB1VTcc706crpVemR4zSXJvanrNZn992vQlEHs27Ec288I3uJXlfNL8kqPcKYw7TVzO1KwqDoypa5yg7qBSwYwPrMjW7PizkyjY7VoSpNcDhojkfFhrfpmOuOxMP1RNUjw5-c6g2iEjh_PM2bJP2BRHHA0SHnfMbP2sPprD5m4nzinpbTzpzKs27QcziS1ZVddcq5rS-aRWkPh1WTw7OikWcfGR_ac2tnxXRR6hFE-vDTTz9kCEHu3_-Mf86tWo-efKJbD9_Xv5883H-sW49-_fALRO9pNYeS0OChRWGSHREuZkwBauhqbK1rqq1yqU_b49m80nZ4AIGHDZQ3vkHWcbcopsPBPedLHgE3JYQ3ndTznoqwn5dHZVPWY3jb8VBuPCeq8V3scjxrLtQwtLSKRuddZPn2WnmRpJdT2iKfXen0SlpxW469IX0x2ZAsxhrrZ7gq7Qw4PccprcOuPO82EJGjavUxplp0R1sZo4rxkmafSTbefbfvZw657Mg5VHY-PJQlPBwcImxVHx6ilHaFPr6nn70Pj2ZT1j6MXNc8josoN57su3HH6ucqdV63neuKdFd3sLecX3fs1Y45uSj7EpudbGVNeVy1nKBUWV52spp_tTNrnKrGDfQ80rHgsB2xJzVtqTy-vuUNa-iXesRK5Ri5SRrgt8o7W_qg7j38Jv8XAa88KDHaxaiFJSg4R_tQZxHjUE7eNtWEkP2SWPafPtW9GNccH-6nQZ2zkfdENH1A2Q6Y88HCViyixuR_HliOX0NNPpbisRwrXh2wJ4jXZsvnsCua47I7vOeYg8WXJ6j1dSTr-Z6jNsVL4-dglvNieqheB10evbCj07I-vjqsUi1TpAApe9JWix9ol5_aMJzVvzz4Q4yyrpP9_EiDDL59yrlYcq3UY6ur44QldM8p_NEom6Tl0eiIVTpK_FGU8JQ4jqJ4nPoRhU1R17OOPDrZEgO0Ozs8xrse_YtudjQtnpU7Ozwruf6wVqU0EKU4T1MCV_UzctEOjxoCj4vxSUn5JyR0RylnhitQPJCo_Ok6kQXXcQS4OS2mHOUGnXcnamhaCXmeY9xUJ0_hzCsOxgTaKo63ZN2AwMOe0Fe9o9msazu8Rm6xnhWRlBJqhuSctUFe_fWDJ93p9MZBdg05eb02zoMSUpkJ_J8hnNLf4R8fCA7matEA2dzQQFtiFNkZoKtkriZ1IVNQUReZGGzrtOwKiRjRHloMde2kqElSs4USCaH4F7pV_XURCDL6D-tT6p7FXPYpmcHzGl0X84stNqSylnwFJu-FBLMu226raC_qcTXrbVMvTucXAvFMFUlOE5V6ZlLKS5AtFZYS6Qi3TSoqjx1em7qJdEp9LhaQsAC-QO6uAo0HaEXHdGBD_nsCERQHS7eKLQjhez1PH_aWDUh4apbqKPgWpikXJHtBJ7Dc-aztFEr3gi6JLebKPoCaSgMChPGm5L-2UbaXeEFFAm8x72Zyzn5H_iGboLXVuYzz3gAMDFeMqw59hI10XMwhsGZSmCnq3QvtMcuf_RNXhsJeyW530p_-CP2wfl42z16UCzSV2MMtz1kP3VKvF9Uco8rrUYHaFpTBt_8No4k8wxJIAAA):

image

**Comment by huong-li-nguyen** [comment]
Can you try to run it locally? I am not sure whether PyCafe pulls the latest CSS files or the old ones from the server?
If that does not work, what's your `dbc` version on hatch?

**Pull request merged**